aiida_pseudo/cli/install.py

```diff
@@ -23,43 +24,56 @@ def cmd_install():
 
 
 @cmd_install.command('family')
-@click.argument('archive', type=types.FileOrUrl(mode='rb'))
+@click.argument('archive_or_folder', type=types.PathOrUrl(exists=True, file_okay=True))
 @click.argument('label', type=click.STRING)
 @options_core.DESCRIPTION(help='Description for the family.')
 @options.ARCHIVE_FORMAT()
-@options.FAMILY_TYPE()
+@options.PSEUDOS_TYPE()
 @options.TRACEBACK()
 @decorators.with_dbenv()
-def cmd_install_family(archive, label, description, archive_format, family_type, traceback):  # pylint: disable=too-many-arguments
-    """Install a standard pseudo potential family from an ARCHIVE on the local file system or from a URL.
+def cmd_install_family(archive_or_folder, label, description, archive_format, pseudos_type, traceback):  # pylint: disable=too-many-arguments
+    """Install a standard pseudo potential family from a FOLDER or an ARCHIVE (on the local file system or from a URL).
 
-    The command will attempt to infer the archive format from the filename extension of the ARCHIVE. If this fails, the
-    archive format can be specified explicitly with the archive format option, which will also display which formats
-    are supported.
+    The command will attempt first to recognize the passed `archive_or_folder` as a folder in the local system. If not,
```

Comment:
```suggestion
    The command will attempt first to recognize the passed ARCHIVE_FOLDER as a folder in the local system. If not,
```
Since these docstrings will be rendered by `click` and are mostly intended for users of the CLI, I prefer to keep a reference to the params by their "metavar" which is formatted by `click`. The reason is that for users in the `--help` it will show as `ARCHIVE_OR_FOLDER` and so this doesn't correspond to `archive_folder`. For developers the inconsistency is less bad because they know that the function argument is turned into all-caps for the help string